packages/firebase_dynamic_links/android/src/main/java/io/flutter/plugins/firebasedynamiclinks/FirebaseDynamicLinksPlugin.java

```diff
@@ -170,7 +170,8 @@ public void onMethodCall(MethodCall call, Result result) {
   private Map<String, Object> getMapFromPendingDynamicLinkData(
       PendingDynamicLinkData pendingDynamicLinkData) {
     Map<String, Object> dynamicLink = new HashMap<>();
-    dynamicLink.put("link", pendingDynamicLinkData.getLink().toString());
+    Uri link = pendingDynamicLinkData.getLink();
+    dynamicLink.put("link", link != null ? link.toString() : null);
```

Comment:
info: [PendingDynamicLinkData#getUrl()](https://firebase.google.com/docs/reference/android/com/google/firebase/dynamiclinks/PendingDynamicLinkData#public-uri-getlink) might return null. We don't want to crash, just return the null link.